### DISCUSSION TITLE
How to extend the 'default' deploy task

### DISCUSSION THREAD

**Discussion by rabol** [deployphp/deployer#3575] [Help needed]
Hi

I'm deploying a laravel project and is using the laravel recipe, but I would like to do something extra

I don't want to 'modify' the recipe/laravel deploy task, but add extra

can I simply add a new deply task in my deploy.php and both will be executed ?

**Comment by antonmedv** [comment]
https://deployer.org/docs/7.x/api#after

**Reply by rabol** [reply]
cool, and I assume it can take an array of tasks?

**Reply by antonmedv** [reply]
https://deployer.org/docs/7.x/tasks#task-grouping
